res_partner_bank_active/views/res_partner_bank.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="res_partner_bank_form_view">
+        <field name="name">res.partner.bank.form (in res_partner_bank_active)</field>
+        <field name="model">res.partner.bank</field>
+        <field name="inherit_id" ref="base.view_partner_bank_form"/>
+        <field name="arch" type="xml">
+
+            <field name="company_id" position="after">
+                <field name="active"/>
```

Comment:
isn't better to follow new v10 approach of an smart-button?

Reply:
There is no "button box" divs in the base view, and I don't think this module should add it.